### DISCUSSION TITLE
Can I change default repository's config file name per renovate instance?

### DISCUSSION THREAD

**Discussion by lictw** [renovatebot/renovate#25509] [Request Help]
### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us what version of Renovate you run.

latest

### If you're self-hosting Renovate, select which platform you are using.

GitLab self-hosted

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

I want to have 2 Renovate instances for single repository, one that will perform just images upgrade with high rate (every 1-2 minutes) and one to upgrade other things (helm charts versions for example, things without automerge), cause the second is time intensive operation (I also use some postUpgrade commands etc) it's hard to achieve high rate for both operations with single instance, can I do it in some way? The first that I imagine is just to have separate config for second instance, but I don't see such option. 

### What you tried so far.

...

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment]
Renovate always searches for repo config files in the same order

**Comment by lictw** [comment]
@rarkins maybe can I with some combination of `unboarding` options to do so second instance will just ignore repository's config and rely only on self bot configuration?

**Reply by rarkins** [reply]
You can look at requireConfig=ignored

**Reply by lictw** [reply]
Tried, second instance is closing all PRs of main instance) So even separate config will not work cause shared context (repository), only something like renovateID can help, seems we can close this.

**Reply by rarkins** [reply]
pruneStalePrs=false will prevent automatic closing of PRs

**Reply by lictw** [reply]
Thanks, this combination (but `pruneStaleBranches`) works and also speeds up the runs, I also found `"dependencyDashboardTitle": "none"` to not touch dashboard.

**Reply by lictw** [reply]
It's a little sad that all this can only be used with automerge cause of `pruneStaleBranches` (so this instance can't clear even own things), but for now it's right that I need.

**Comment by lictw** [comment]
I also can't have schedule per packageRule, right? For ex. schedule for whole repo like "once in a week" but "null" for my images, so every run.

**Reply by rarkins** [reply]
You can, but careful not to have multiple schedules within a single branch/PR because there can be only one which "wins"
